File: LocalHiringPlatform.Domain/Entities/Job.cs
using LocalHiringPlatform.Domain.Entities;

public class Job : BaseEntity
{
    public Guid EmployerProfileId { get; set; }

    public EmployerProfile EmployerProfile { get; set; } = null!;

    public string Title { get; set; } = string.Empty;

    public string Description { get; set; } = string.Empty;

    public string City { get; set; } = string.Empty;

    public string State { get; set; } = string.Empty;

    public decimal? MinSalary { get; set; }

    public decimal? MaxSalary { get; set; }

    public int ExperienceRequired { get; set; }

    public string? RequiredSkills { get; set; }

    public bool IsActive { get; set; } = true;
}

File: UI/localhiringplatform-ui/src/types/Job.ts
export interface Job {
    entityId: string;

    title: string;

    description: string;

    city: string;

    state: string;

    minSalary?: number;

    maxSalary?: number;

    experienceRequired: number;

    maxExperienceRequired: number;

    requiredSkills?: string;

    isActive: boolean;

    applicantCount?: number;
}